cmd/main.go

```diff
@@ -3,9 +3,50 @@ package main
 import (
 	"log"
 	"time"
+
+	"github.com/lvl484/positioning-filter/config"
+	"github.com/lvl484/positioning-filter/storage"
+)
+
+const (
+	configPath = "../config"
+	configName = "viper.config"
 )
 
 func main() {
+	viper, err := config.NewViperCfg(configName, configPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// CONSUL ------------------------
+	consulCfg := viper.NewConsulConfig()
+	agentCfg := consulCfg.AgentConfig()
+	clientCfg, err := consulCfg.NewClient()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err = consulCfg.ServiceRegister(clientCfg, agentCfg); err != nil {
+		log.Fatal(err)
+	}
+
+	// -------------------------------
+
+	// POSTGRES ----------------------
```

Comment:
Don't leave such comments in code, if you want to separate code blocks - move them to functions/structures/packages.

Reply:
like init() ?

Reply:
i can remove comments. Its just to increase readability

Reply:
removed comments